### PR TITLE
929 update content

### PR DESCRIPTION
**Pull request by goodviber**
### Context
We need to update the content in Prod to match the prototype. We are changing the line of content that appears above the locations list in the 'School placement' section and the content that appears in the GIT 'Where you will train' box when the provider is a SCITT.

Because we are changing the content on Find, we'll also need to change it in the preview in Publish (see attached screenshot).

See attached card and let me know if you have any questions :)
Tech notes

Make sure to have the counter work for new find in publish

### Changes proposed in this pull request
2 file content changes
### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by goodviber** [comment]
[Change 'School placements' and 'Where you will train' content](https://trello.com/c/ivK8AKLO/950-change-school-placements-and-where-you-will-train-content)

File: app/components/find/courses/about_schools_component/view.html.erb [hunk 1]
```diff
@@ -23,7 +22,7 @@
 
     <% if course.site_statuses.map(&:site).uniq.many? %>
       <h3 class="govuk-heading-m">Locations</h3>
-      <p class="govuk-body">You can select one of the following locations when you apply.</p>
+      <p class="govuk-body">You can select one of the following locations when you apply to indicate a preferred placement.</p>
```
Comment (avinhurry) [inline app/components/find/courses/about_schools_component/view.html.erb:25]: Did the content change? In the proto it's: 

```suggestion
      <p class="govuk-body">You can select one of the following locations to indicate a preferred placement when you apply.

</p>
```

File: app/views/publish/courses/course_information/edit.html.erb [hunk 2]
```diff
@@ -106,7 +106,8 @@
       <% elsif @course.program_type == 'scitt_programme' && @provider.provider_code != 'E65' %>
         <%= govuk_details(summary_text: "See the guidance we show in this section") do %>
           <h3 class="govuk-heading-m">Where you will train</h3>
-          <p class="govuk-body">You’ll be placed in different schools during your training. You cannot pick which schools you want to be in, but your course will try to place you in schools you can commute to.</p>
+          <p class="govuk-body">You'll be placed in different schools during your training. You can't pick which schools you want to be in, but your training provider will place you in schools you can travel to.</p>
```
Comment (avinhurry) [inline app/views/publish/courses/course_information/edit.html.erb:109]:
```suggestion
          <p class="govuk-body">You’ll be placed in different schools during your training. You can’t pick which schools you want to be in, but your training provider will place you in schools you can travel to.</p>
```

File: spec/components/find/courses/about_schools_component/view_spec.rb [hunk 3]
```diff
@@ -105,7 +105,7 @@
 
       result = render_inline(described_class.new(course))
 
-      expect(result.text).to include("You’ll be placed in different schools during your training.")
+      expect(result.text).to include("You'll be placed in different schools during your training.")
```
Comment (avinhurry) [inline spec/components/find/courses/about_schools_component/view_spec.rb:108]:
```suggestion
      expect(result.text).to include("You’ll be placed in different schools during your training.")
```

**Review by avinhurry — CHANGES_REQUESTED** [review]
Can we use and put back the smart quotes please?

**Comment by avinhurry** [comment]
image

Do we want to add that link?

File: app/components/find/courses/about_schools_component/view.html.erb (hunk 1 above)
Comment (goodviber) [reply app/components/find/courses/about_schools_component/view.html.erb:25]: I think so, @victoriawinlow is reviewing the content.

Comment (Kateharries) [inline app/components/find/courses/about_schools_component/view.html.erb:25]: You can select one of the following locations to indicate a preferred placement when you apply.

**Review by Kateharries — COMMENTED** [review]
I've just added a comment on the line that needs to be updated.

**Review by avinhurry — APPROVED** [review]

**Pull request merged**